lib/smarter_csv/smarter_csv.rb

```diff
@@ -186,6 +186,10 @@ def headers
       @headers
     end
 
+    def headers_count
+      @headers_count
```

Comment:
this won't work.
Did you mean:

```
   @headers.count
```

In any case, users can just simply call `.count` in their code themselves when accessing `headers`